Build fails on stackage nightly

```
       While building package krank-0.3.0 (scroll up to its section to see the error) using:
       /var/stackage/.stack/setup-exe-cache/x86_64-linux-tinfo6/Cabal-simple_6HauvNHV_3.10.1.0_ghc-9.6.2 --verbose=1 --builddir=.stack-work/dist/x86_64-linux-tinfo6/Cabal-3.10.1.0 build lib:krank exe:krank test:krank-test --ghc-options ""
       Process exited with code: ExitFailure 1 
       Logs have been written to: /var/stackage/work/unpack-dir/.stack-work/logs/krank-0.3.0.log
       
       Configuring krank-0.3.0...
       Preprocessing test suite 'krank-test' for krank-0.3.0..
       Building test suite 'krank-test' for krank-0.3.0..
       [ 1 of 13] Compiling Krank.Types
       [ 2 of 13] Compiling Krank.Checkers.Ignore
       [ 3 of 13] Compiling Utils.Display
       
       /var/stackage/work/unpack-dir/unpacked/krank-0.3.0-4ff61633f1bf9efb948a9f48699af6a02aa0828742419d452b89b388805bf1d5/src/Utils/Display.hs:18:5: warning: [GHC-62161] [-Wincomplete-uni-patterns]
           Pattern match(es) are non-exhaustive
           In a pattern binding:
               Patterns of type ‘[Text.Text]’ not matched: []
          |
       18 |     (firstLine : nextLines) = Text.splitOn "\n" text
          |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
       [ 4 of 13] Compiling Krank.Formatter
       [ 5 of 13] Compiling Utils.Req
       [ 6 of 13] Compiling Utils.Gitlab
       [ 7 of 13] Compiling Utils.Github
       [ 8 of 13] Compiling Krank.Checkers.IssueTracker
       [ 9 of 13] Compiling Krank
       [11 of 13] Compiling Test.Utils.GitlabSpec
       
       /var/stackage/work/unpack-dir/unpacked/krank-0.3.0-4ff61633f1bf9efb948a9f48699af6a02aa0828742419d452b89b388805bf1d5/src/Krank.hs:49:3: error: [GHC-88464]
           Variable not in scope:
             forM_
               :: [Either Text.Text [Violation]]
                  -> (Either Text.Text (t0 Violation) -> m0 ()) -> m a0
          |
       49 |   forM_ res $ \case
          |   ^^^^^
       [12 of 13] Compiling Test.Utils.GithubSpec
```

You can open a stackage PR to re-enable when ready.

--- a/.envrc
+++ b/.envrc
@@ -1,4 +1,1 @@
-use nix
-watch_file default.nix
-watch_file nixpkgs.nix
-watch_file shell.nix
+use flake

--- a/app/Main.hs
+++ b/app/Main.hs
@@ -21,6 +21,7 @@ import System.Exit (exitFailure)
 import System.Process
 import Text.Regex.PCRE.Heavy
 import Version (displayVersion)
+import Control.Monad (unless)
 
 data KrankOpts = KrankOpts
   { codeFilePaths :: [FilePath],

--- a/src/Krank.hs
+++ b/src/Krank.hs
@@ -25,6 +25,7 @@ import Krank.Types
 import qualified Network.HTTP.Req as Req
 import PyF
 import System.IO (stderr)
+import Control.Monad (forM_)
 
 processFile ::
   MonadKrank m =>

